Test/AddCompIonsTest.cs

```diff
@@ -139,7 +139,7 @@ public static void TestCompIons_ModernSearch()
 
             Assert.AreEqual(allPsmsArray[0].ScanNumber, allPsmsArray2[0].ScanNumber);
 
-            Assert.IsTrue(allPsmsArray2[0].Score < allPsmsArray[0].Score * 2 && allPsmsArray2[0].Score + 2 > allPsmsArray[0].Score * 2);
+            Assert.IsTrue(allPsmsArray2[0].Score < allPsmsArray[0].Score * 2 && allPsmsArray2[0].Score > allPsmsArray[0].Score + 3);
```

Comment:
Recalculation of score bumped a decimal over the tolerance, but I think it's a better way to do it, so updated unit test